google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/snippets/JobSnippets.java

```diff
@@ -89,10 +104,55 @@ public boolean waitForWithOptions() throws InterruptedException {
       } else {
         // job completed successfully
       }
-      // [END waitFor]
+      // [END waitForWithOptions]
     } catch (TimeoutException e) {
       return true;
     }
     return true;
   }
+
+  /**
+   * Example of reloading all fields until job status is DONE.
+   */
+  // [TARGET reload(JobOption...)]
+  public JobStatus.State reload() throws InterruptedException {
+    Job job = this.job;
```

Comment:
Because `this.job` is declared `final`. (I was following the conventions from the LoggingSnippets class) The `reload()` call does not mutate the Job object, so I need to reassign job to get the latest values.